apps/prairielearn/src/sprocs/authz_course_instance.sql

```diff
@@ -91,8 +81,6 @@ BEGIN
 
     permissions_course_instance := jsonb_build_object(
         'course_instance_role', course_instance_role,
-        'has_course_instance_permission_view', course_instance_role >= 'Student Data Viewer',
```

Comment:
This was replaced by a helper in `authz-data.ts`.

Reply:
Is there a reason you didn't make the same change in `authz_course`?